Changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.4
+### Added
+- `rlpBytesKeccak256(RLPItem)` returns keccak256 hash of RLP encoded bytes.
+- `dataKeccak256(RLPItem)` returns keccak256 hash of the unerlying data.
```

Comment:
small typo s/unerlying/underlying

Reply:
will fix this and update the name tomorrow